pytition/pytition/wsgi.py

```diff
@@ -8,9 +8,45 @@
 """
 
 import os
+import sys
 
 from django.core.wsgi import get_wsgi_application
 
 os.environ.setdefault("DJANGO_SETTINGS_MODULE", "pytition.settings")
 
 application = get_wsgi_application()
+
+if os.environ.get('EMAIL_BACKEND') == 'mailer':
+    from django.conf import settings
```

Comment:
It does not seem easy to add environment variables to the WSGI environment from, say, the apache config. (no, SetEnv does not work :/)

I propose to just use settings for that.

I propose to 
* move the ```from django.conf import settings``` on line 16.
* Modify line 19 to read ```if settings.EMAIL_BACKEND == 'mailer':```

That would make sense since ```EMAIL_BACKEND``` is a Django config anyway https://docs.djangoproject.com/fr/2.2/ref/settings/#std:setting-EMAIL_BACKEND

What do you think?

Reply:
yes it seems a better way to do this!